packages/ruleset-migrator/src/__tests__/__fixtures__/functions-variant-4/output.cjs

```diff
@@ -0,0 +1,18 @@
+const { oas3: oas3, oas3_1: oas3_1 } = require('@stoplight/spectral-formats');
+const oas3$0 = _interopDefault(require('/.tmp/spectral/functions-variant-4/functions/oas3.js'));
```

Comment:
before the change, you'd get
```ts
const oas3 = 
```
which presumably could lead to an error if an OAS format was used too.